xml/System.Linq/Enumerable.xml

```diff
@@ -13407,12 +13634,26 @@ If `count` is not a positive number, this method returns an empty enumerable col
         <Parameter Name="count" Type="System.Int32" RefType="out" Index="1" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <typeparam name="TSource">To be added.</typeparam>
-        <param name="source">To be added.</param>
-        <param name="count">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TSource">The type of the elements of <paramref name="source" />.</typeparam>
+        <param name="source">A sequence that contains elements to be counted.</param>
+        <param name="count">When this method returns, contains the count of <paramref name="source" /> if successful, or zero if the method failed to determine the count.</param>
+        <summary>Attempts to determine the number of elements in a sequence without forcing an enumeration.</summary>
+        <returns>
+          <see langword="true" /> if the count of <paramref name="source" /> can be determined without enumeration; otherwise, <see langword="false" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The method performs a series of type tests, identifying common subtypes whose
```

Comment:
Seeing indentation issues here again. I presume it's not a big issue since it's markdown?

Reply:
It's not an issue, especially in remarks. MS Docs will trim any prefixed whitespace.